src/Server.cpp

```diff
@@ -220,6 +220,25 @@ bool cServer::InitServer(cSettingsRepositoryInterface & a_Settings, bool a_Shoul
 		LOGWARNING("WARNING: BungeeCord is allowed and server set to online mode. This is unsafe and will not work properly. Disable either authentication or BungeeCord in settings.ini.");
 	}
 
+	int Versions[] =
+	{
+		// TODO: better way to get all supported versions from cProtocolRecognizer?
+		cProtocolRecognizer::PROTO_VERSION_1_8_0,
+		cProtocolRecognizer::PROTO_VERSION_1_9_0,
+		cProtocolRecognizer::PROTO_VERSION_1_9_1,
+		cProtocolRecognizer::PROTO_VERSION_1_9_2,
+		cProtocolRecognizer::PROTO_VERSION_1_9_4,
+		cProtocolRecognizer::PROTO_VERSION_1_10_0,
+		cProtocolRecognizer::PROTO_VERSION_1_11_0,
+		cProtocolRecognizer::PROTO_VERSION_1_11_1,
+		cProtocolRecognizer::PROTO_VERSION_1_12,
+	};
+	for (size_t i = 0; i < sizeof(Versions) / sizeof(Versions[0]); ++i)
+	{
+		cForgeMods Mods;
+		m_ForgeModsByVersion.insert({Versions[i], Mods});
+	}
```

Comment:
No need to pre-initialize this, just add a check to the registration function so that it creates the map entry if it doesn't exist yet.